hazelcast-jet-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/generator/SqlStreamGeneratorTest.java

```diff
@@ -112,8 +112,8 @@ public void when_coercionIsRequired_then_throws() {
 
     @Test
     public void test_nullArgument() {
-        assertThatThrownBy(() -> sqlService.execute("SELECT * FROM TABLE(GENERATE_STREAM(null))"))
-                .hasMessage("rate cannot be null");
+        assertThatThrownBy(() -> sqlService.execute("SELECT * FROM TABLE(GENERATE_STREAM(null))").iterator().next())
+                .hasMessageContaining("rate cannot be null");
```

Comment:
Will the user know that this applies to the `GENERATE_STREAM` function? Imagine this function is in a huge query. I'd like to see it asserted by this test.

Same for other errors in this PR.